package.json

```diff
@@ -127,10 +140,10 @@
     "esm": "=3.2.25",
     "expect": "=27.4.2",
     "express": "^4.17.1",
-    "file-loader": "^6.0.0",
     "git-describe": "^4.1.0",
+    "html-loader": "^3.0.1",
```

Comment:
It seems that `html-loader` is not needed in build system. I've removed referenced of it in webpack config and the build system worked the same as before as `html-webpack-plugin` is handling what it should.

I've left comments in appropriate configs where this plugin is used.

Reply:
removed.